shell/agents/Microsoft.Azure.Agent/AzureAgent.cs

```diff
@@ -110,11 +111,34 @@ public void Initialize(AgentConfig config)
                 .CreateLogger();
             Log.Information("Azure agent initialized.");
         }
+
+        if (_setting.Telemetry)
```

Comment:
@NoriZC FYI, we now support setting file. You can open the default setting file by typing `/agent config` while using the `azure` agent. It now only supports configure `telemetry` and `logging`. So, we use that to control the enabling/disabling the telemetry.